### DISCUSSION TITLE
Models not shown in the dropdowns

### DISCUSSION THREAD

**Discussion by michael-pyne** [AltimateAI/vscode-dbt-power-user#293] [General]
Hi. 

I am trying to figure out how to get my models shown in the drop-downs on the left here. Does anyone know what I am doing wrong? Everything else works (query previews etc.) 

<img width="354" alt="image" src="https://user-images.githubusercontent.com/123483857/214344817-a229d131-191c-4236-808a-1515c40a42be.png">

**Comment by GeorgeHC97** [comment]
Having the same issue. Clicking to run parent models runs the dbt command correctly but no models are shown in the sidebar.
